src/reference_implementations/prompting_vector_llms/activation_fine_tuning/compute_activations.ipynb

```diff
@@ -87,31 +87,505 @@
+       " 'layers.34.attention.wk',\n",
+       " 'layers.34.attention.wv',\n",
+       " 'layers.34.attention.wo',\n",
+       " 'layers.34.feed_forward',\n",
+       " 'layers.34.feed_forward.w1',\n",
+       " 'layers.34.feed_forward.w2',\n",
+       " 'layers.34.feed_forward.w3',\n",
+       " 'layers.34.attention_norm',\n",
+       " 'layers.34.ffn_norm',\n",
+       " 'layers.35',\n",
+       " 'layers.35.attention',\n",
+       " 'layers.35.attention.wq',\n",
+       " 'layers.35.attention.wk',\n",
+       " 'layers.35.attention.wv',\n",
+       " 'layers.35.attention.wo',\n",
+       " 'layers.35.feed_forward',\n",
+       " 'layers.35.feed_forward.w1',\n",
+       " 'layers.35.feed_forward.w2',\n",
+       " 'layers.35.feed_forward.w3',\n",
+       " 'layers.35.attention_norm',\n",
+       " 'layers.35.ffn_norm',\n",
+       " 'layers.36',\n",
+       " 'layers.36.attention',\n",
+       " 'layers.36.attention.wq',\n",
+       " 'layers.36.attention.wk',\n",
+       " 'layers.36.attention.wv',\n",
+       " 'layers.36.attention.wo',\n",
+       " 'layers.36.feed_forward',\n",
+       " 'layers.36.feed_forward.w1',\n",
+       " 'layers.36.feed_forward.w2',\n",
+       " 'layers.36.feed_forward.w3',\n",
+       " 'layers.36.attention_norm',\n",
+       " 'layers.36.ffn_norm',\n",
+       " 'layers.37',\n",
+       " 'layers.37.attention',\n",
+       " 'layers.37.attention.wq',\n",
+       " 'layers.37.attention.wk',\n",
+       " 'layers.37.attention.wv',\n",
+       " 'layers.37.attention.wo',\n",
+       " 'layers.37.feed_forward',\n",
+       " 'layers.37.feed_forward.w1',\n",
+       " 'layers.37.feed_forward.w2',\n",
+       " 'layers.37.feed_forward.w3',\n",
+       " 'layers.37.attention_norm',\n",
+       " 'layers.37.ffn_norm',\n",
+       " 'layers.38',\n",
+       " 'layers.38.attention',\n",
+       " 'layers.38.attention.wq',\n",
+       " 'layers.38.attention.wk',\n",
+       " 'layers.38.attention.wv',\n",
+       " 'layers.38.attention.wo',\n",
+       " 'layers.38.feed_forward',\n",
+       " 'layers.38.feed_forward.w1',\n",
+       " 'layers.38.feed_forward.w2',\n",
+       " 'layers.38.feed_forward.w3',\n",
+       " 'layers.38.attention_norm',\n",
+       " 'layers.38.ffn_norm',\n",
+       " 'layers.39',\n",
+       " 'layers.39.attention',\n",
+       " 'layers.39.attention.wq',\n",
+       " 'layers.39.attention.wk',\n",
+       " 'layers.39.attention.wv',\n",
+       " 'layers.39.attention.wo',\n",
+       " 'layers.39.feed_forward',\n",
+       " 'layers.39.feed_forward.w1',\n",
+       " 'layers.39.feed_forward.w2',\n",
+       " 'layers.39.feed_forward.w3',\n",
+       " 'layers.39.attention_norm',\n",
+       " 'layers.39.ffn_norm',\n",
+       " 'norm',\n",
+       " 'output']"
       ]
      },
-     "execution_count": 7,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "model.module_names[-10:]"
+    "model.module_names"
```

Comment:
Looks like we're listing all the modules instead of the last 10 as described above?

Reply:
Yes, good call. I'll modify the comment.